.github/workflows/ci.yml

```diff
@@ -20,14 +20,17 @@ jobs:
     name: "${{ matrix.compiler }} (Boost: ${{ matrix.boost }})"
     steps:
       - uses: actions/checkout@master
+      - name: update package list
+        run: apt-get update
```

Comment:
Doesn't the docker image already contain more or less up-to-date package lists?

(General question)

Reply:
Well... yes that's the case but we don't know how up-to-date it is. I ran into that issue when bumping the boost dependency. IMO it's better to keep the image small (remove `/var/lib/apt/lists/*`) and do a fresh update on demand.